docs/reference/commandline/run.md

```diff
@@ -242,6 +243,14 @@ Docker daemon.
 
 For in-depth information about volumes, refer to [manage data in containers](../../tutorials/dockervolumes.md)
 
+### Mount using the New API (--mount)
+
+The new mount api is described in the [service create command section](service_create.md).
+
+For the last bind example, the command line would be:
+
+    $ docker run -t -i --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock busybox sh
```

Comment:
Let's use volumes as the primary example of mounts. Binds should be a special case.

This example is salient, as well, so using this with a volume example will demonstrate the difference in behavior of volumes and bind.